specification.md

```diff
@@ -3555,12 +3555,14 @@ The `scheme` property defines the auth scheme to be used. Can be "bearer", "basi
 The `properties` property defines the auth scheme properties information.
 Can be one of ["Basic properties definition"](#basic-properties-definition), ["Bearer properties definition"](#bearer-properties-definition), or ["OAuth2 properties definition"](#oauth2-properties-definition)
 
+Implementations might impose restrictions on the expressions that might be used to resolve auth properties. 
```

Comment:
```suggestion
Implementations might impose restrictions on the expressions used to resolve auth properties depending on the context used. 
```

I think we need to be more specific here. For example, we could argue that an implementation could restrict some expressions for security reasons. I'm on the fence; to me, this is still implementation-specific. Not sure if we need to explicitly have this here.

Reply:
I think is ok to mention in spec those places where there is implementor discrection.